Description: Fix missing Openssl MD5_FINAL symbol
 Fix for issue #3632. Allows package build
Author: Jean Baptiste Favre <debian@jbfavre.org>
Origin: other, https://issues.apache.org/jira/browse/TS-3632
Forwarded: https://github.com/apache/trafficserver/pull/203
Bug: https://issues.apache.org/jira/browse/TS-3632
Last-Update: 2015-05-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: cmd/traffic_wccp/Makefile.am
===================================================================
diff --git a/cmd/traffic_wccp/Makefile.am b/cmd/traffic_wccp/Makefile.am
index 949388c..f7b6148 100644
--- a/cmd/traffic_wccp/Makefile.am
+++ b/cmd/traffic_wccp/Makefile.am
@@ -22,7 +22,8 @@ AM_CPPFLAGS = $(iocore_include_dirs) \
   -I$(top_srcdir)/lib \
   -I$(top_srcdir)/lib/records \
   -I$(top_srcdir)/lib/ts \
-  -I$(top_srcdir)/lib/wccp
+  -I$(top_srcdir)/lib/wccp \
+  @OPENSSL_INCLUDES@
 
 AM_LDFLAGS = \
   @EXTRA_CXX_LDFLAGS@ \
@@ -37,5 +38,5 @@ traffic_wccp_SOURCES = \
 traffic_wccp_LDADD = \
   $(top_builddir)/lib/tsconfig/libtsconfig.la \
   $(top_builddir)/lib/wccp/libwccp.a \
-  $(top_builddir)/lib/ts/libtsutil.la
-
+  $(top_builddir)/lib/ts/libtsutil.la \
+  @OPENSSL_LIBS@
